### PR TITLE
Network map

### PR DESCRIPTION
**Pull request by jebonfig**

### CODE REVIEW

File: src/modules/network-map/registration.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import DomainIcon from 'mdi-react/DomainIcon';
+import useModuleTranslation from '../../core/hooks/useModuleTranslation';
+import i18n from 'i18next';
+import enData from './translations/en.json';
+import WebIcon from 'mdi-react/WebIcon';
+import { Dashboard } from './views/Dashboard';
+import { Organizations } from './views/Organizations';
+import { Nodes } from './views/Nodes';
+
+const NETWORK_MAP_ROUTE_PREFIX = '/network-map';
```
Comment (eberger727) [inline src/modules/network-map/registration.ts:29]: should this be `/namespace/:namespace/network-map`?

File: src/modules/network-map/views/Dashboard.tsx [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+      : undefined;
+    const node = isNode
+      ? nodes?.find((n) => n.id === p.node.id) || {
+          name: p.label,
+        }
+      : undefined;
+    return (
+      <>
+        <text
+          // onClick={() => alert('label clicked')}
```
Comment (eberger727) [inline src/modules/network-map/views/Dashboard.tsx:68]: maybe remove this, unless it's needed for future demos

File: src/modules/network-map/views/Dashboard.tsx [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+  const circleComponent: (props: CircleProps<DATUM>) => JSX.Element = (p) => {
+    const isOrg = p.node.depth === 1;
+    const isNode = p.node.depth === 2 && p.node.value === NODE_VALUE;
+    const isIdentity = p.node.depth === 2 && p.node.value === IDENTITY_VALUE;
+    let fill = '';
+    // eslint-disable-next-line
+    let onClick = () => {};
+    if (isOrg) {
+      fill = "url('#orgGradient')";
+      // onClick = () => alert('org circle clicked');
```
Comment (eberger727) [inline src/modules/network-map/views/Dashboard.tsx:115]: these too

File: src/modules/network-map/registration.ts (hunk 1 above)
Comment (jebonfig) [reply src/modules/network-map/registration.ts:29]: If the module uses namespaces yes, but the network map doesn't

**Review by eberger727 — APPROVED** [review]

**Pull request merged**